notifications/11-29-22-0900.md

```diff
@@ -0,0 +1,29 @@
+**What is happening?**
+
+Due to the size of the clusters and some potential impact on etcd we are going to reduce the retention of builds, images and deployments to 48 hours from 96 hours. This should help maintain the performance of etcd and reduce the retention of older images, builds and deployments.
+
+There are 2 scheduled changes:
+
+SILVER/EMERALD - December 6th @ 0900 - CHG0043455
+GOLD/GOLDDR - December 7th @ 0900 - CHG0043457
+
+**When?**
+
+Dates listed above with the individual changes.
+
+**Will there be an impact on the Platform apps?**
+
+In each change, the change will be applied and then tested. The changes should be entirely transparent and not cause any outages or disruptions. The change can be quickly reverted if there's any issues discovered during implementation.
```

Comment:
```suggestion
In each change, the change will be applied and then tested. The changes should be entirely transparent and not cause any outages or disruptions. The change can be quickly reverted if there's any issues discovered during implementation.

BuildConfigs will be changed to only keep the most recent 2 builds instead of 5 OR builds within the past 48 hours, instead of the past 96 hours. This will reduce the number of old builds you see in your tools namespaces.

DeploymentConfigs will be changed to only keep the most recent 2 deploy pods instead of 5 OR deploys within the past 48 hours, instead of the past 96 hours. This will reduce the number of old deploy pods you see in your namespaces.
```